packages/e2e/test/wallet/PersonalWallet/handle.test.ts

```diff
@@ -0,0 +1,252 @@
+    const keyHash = await keyAgent.bip32Ed25519.getPubKeyHash(pubKey);
+    policySigner = new util.KeyAgentTransactionSigner(keyAgent, derivationPath);
+    policyScript = {
+      __type: Cardano.ScriptType.Native,
+      kind: Cardano.NativeScriptKind.RequireAllOf,
+      scripts: [
+        {
+          __type: Cardano.ScriptType.Native,
+          keyHash,
+          kind: Cardano.NativeScriptKind.RequireSignature
+        }
+      ]
+    };
+    policyId = nativeScriptPolicyId(policyScript);
+    wallet.shutdown();
+  };
+
+  const mint = async (tokens: Cardano.TokenMap, txMetadatum: Cardano.Metadatum) => {
+    const auxiliaryData = {
+      blob: new Map([[721n, txMetadatum]])
+    };
+
+    const txProps: InitializeTxProps = {
+      auxiliaryData,
+      mint: tokens,
+      outputs: new Set([
+        {
+          address: walletAddress,
+          value: {
+            assets: tokens,
+            coins
+          }
+        }
+      ]),
+      witness: { extraSigners: [policySigner], scripts: [policyScript] }
+    };
+
+    const unsignedTx = await wallet.initializeTx(txProps);
+
+    const finalizeProps: FinalizeTxProps = {
+      auxiliaryData,
+      tx: unsignedTx,
+      witness: { extraSigners: [policySigner], scripts: [policyScript] }
+    };
+
+    const signedTx = await wallet.finalizeTx(finalizeProps);
+    await submitAndConfirm(wallet, signedTx);
+  };
+
+  const restartWallet = async () => {
+    wallet.shutdown();
+    wallet = (
+      await getWallet({
+        env,
+        handlePolicyIds: [policyId],
+        idx: 0,
+        logger,
+        name: 'Minting Wallet',
+        polling: { interval: 50 }
+      })
+    ).wallet;
+    await walletReady(wallet, coins);
+  };
+
+  beforeAll(async () => {
+    await initPolicyId();
+    wallet = (
+      await getWallet({
+        env,
+        handlePolicyIds: [policyId],
+        idx: 0,
+        logger,
+        name: 'Minting Wallet',
+        polling: { interval: 50 }
+      })
+    ).wallet;
+    await walletReady(wallet, coins);
+    receivingWallet = (
+      await getWallet({
+        env,
+        handlePolicyIds: [policyId],
+        idx: 1,
+        logger,
+        name: 'Receiving Wallet',
+        polling: { interval: 50 }
+      })
+    ).wallet;
+    await walletReady(wallet, coins);
+
+    assetIds = [Cardano.AssetId(`${policyId}${assetNames[0]}`), Cardano.AssetId(`${policyId}${assetNames[1]}`)];
+    walletAddress = (await firstValueFrom(wallet.addresses$))[0].address;
+  });
+
+  afterEach(async () => {
+    await burnTokens({
+      policySigners: [policySigner],
+      scripts: [policyScript],
+      wallet
+    });
+    await burnTokens({
+      policySigners: [policySigner],
+      scripts: [policyScript],
+      wallet: receivingWallet
+    });
+  });
+
+  afterAll(async () => {
+    wallet.shutdown();
+    receivingWallet.shutdown();
+  });
+
+  it('mint and send handle', async () => {
```

Comment:
```suggestion
  it('PersonalWallet discovers it's own handles', async () => {
```
We're not testing the minting of handles